### ISSUE TITLE
Adding alt text to image on account page

### ISSUE THREAD

**Issue by jpb0042** [WVU-CS230-2022-08-Group10/Logique-a-la-Crabe-2.0#1]
The image used in the popup after a user creates an account does not have alt text. This would be necessary if the image does not load properly.

- img src = "assets/images/and_gate.jpg" style="width:100px;height:100px" - alt text should've gone here>

John Benedickson
